File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/Should_Add_Skip_Compatibility_Flag_To_Arguments_If_Set.cs
using Xunit;

namespace DefaultNamespace
{
    public class ChocolateyUpgraderTests
    {
        [Fact(Skip = "")]
        public void Should_Add_Skip_Compatibility_Flag_To_Arguments_If_Set()
        {
            {
                // Given
                var fixture = new ChocolateyUpgraderFixture();
                fixture.Settings.SkipCompatibilityChecks = skipCompatibiity;
                // When
                var result = fixture.Run();
                // Then
                Assert.Equal(expected, result.Args);
            }
        }
    }
}

File: ignored-test/java-junit/Should_Add_Skip_Compatibility_Flag_To_Arguments_If_Set.java
import org.junit.Test;
import org.junit.Ignore;
import static org.junit.Assert.assertEquals;

public class ChocolateyUpgraderTests {

    @Test
    @Ignore("") 
    public void shouldAddSkipCompatibilityFlagToArgumentsIfSet() {
        // Given
        ChocolateyUpgraderFixture fixture = new ChocolateyUpgraderFixture();
        fixture.Settings.SkipCompatibilityChecks = skipCompatibiity;
        // When
        var result = fixture.Run();
        // Then
        assertEquals(expected, result.Args);
    }
}
